frontend/src/components/ClientGallery.tsx

```diff
@@ -15,9 +15,13 @@ export default function ClientGallery(props: ClientGalleryProps) {
     })
 
     return (
-        <div className="client-gallery">
-            <Link to="/clients/add">NEUER KUNDE</Link>
-            {clientCards}
-        </div>
+        <Grid container direction='column' alignItems='center'>
```

Comment:
This works. If you need more flexibility <Box/> with sx={{display: flex, flexDirection: column}} is another possible solution.